ogr/ogrct.cpp

```diff
@@ -2325,7 +2356,7 @@ int OGRProjCT::TransformWithErrorCodes(int nCount, double *x, double *y,
     {
         if (m_eSourceFirstAxisOrient == OAO_East)
         {
-            for (int i = 0; i < nCount; i++)
+            for (size_t i = 0; i < nCount; i++)
             {
                 if (x[i] != HUGE_VAL && y[i] != HUGE_VAL)
```

Comment:
Out of scope but I was wondering if in general we could replace HUGE_VAL with std::numeric_limit<>::infinity.

Reply:
I keep HUGE_VAL, as a way of connecting with the convention used in PROJ source code.